app/components/RecentEvents.m.less

```diff
@@ -101,6 +105,7 @@
   opacity: 0.7;
   overflow: hidden;
   text-overflow: ellipsis;
+  white-space: normal !important;
```

Comment:
What rule is this overriding?  Something global?

Reply:
`.cell span` which beats it in specificity slightly but applies to every other span in the cell